stack.yaml

```diff
@@ -42,14 +42,18 @@ extra-deps:
 - optparse-applicative-0.15.1.0
 - ormolu-0.1.2.0
 - parser-combinators-1.2.1
+- primitive-0.7.1.0
 - regex-base-0.94.0.0
 - regex-pcre-builtin-0.95.1.1.8.43
 - regex-tdfa-1.3.1.0
 - retrie-0.1.1.1
 - semialign-1.1
+# - github: wz1000/shake
```

Comment:
probably, @alanz commented out it a while ago
lets merge this as is, to avoid problems to stack users, we can clean all the stack yamls in the next version bump